stable/optimus/templates/data-cleanup-cronjob.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: operator-run-cleanup
+spec:
+  schedule: "0 0 1 * *"  # Runs At 00:00 on day-of-month 1
```

Comment:
Let's discuss how to read it from values.

Reply:
cool